### PR TITLE
Update plugins recipes with the plugins prefix.

### PR DESCRIPTION
**Pull request by godofredoc**
The plugins recipes are inside a plugins folder that was missing in the
build definitions.

### CODE REVIEW

File: config/plugins_config.star [hunk 1]
```diff
@@ -97,13 +97,14 @@ def plugins_prod_config(branch, version, testing_ref, release_ref, platform_args
...
     """
-    recipe_name = ("plugins_" + version if version else "plugins")
-    luci.recipe(
-        name = recipe_name,
-        cipd_package = "flutter/recipe_bundles/flutter.googlesource.com/recipes",
-        cipd_version = "refs/heads/master",
-        use_bbagent = True,
-    )
+    recipe_name = ("plugins/plugins_" + version if version else "plugins/plugins")
+    if branch != "master":
```
Comment (yusufm) [inline config/plugins_config.star:101]: Does branch = master need to be handled somewhere?

**Review by yusufm — APPROVED** [review]
LGTM with one comment

File: config/plugins_config.star (hunk 1 above)
Comment (keyonghan) [inline config/plugins_config.star:101]: Is this for non master branch?
(https://ci.chromium.org/p/flutter/g/plugins/console is for master).

Comment (godofredoc) [reply config/plugins_config.star:101]: It is handled in plugins_define_recipes above.

Comment (godofredoc) [reply config/plugins_config.star:101]: No this is only for master. Plugins only supports master.

**Pull request merged**